src/ansible_navigator/actions/settings.py

```diff
@@ -0,0 +1,201 @@
+"""The ``settings`` subcommand action."""
+
+import curses
+
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Tuple
+
+from .._yaml import human_dump
+from ..action_base import ActionBase
+from ..action_defs import RunStdoutReturn
+from ..app_public import AppPublic
+from ..configuration_subsystem import transform_settings
+from ..steps import Step
+from ..ui_framework import CursesLinePart
+from ..ui_framework import CursesLines
+from ..ui_framework import Interaction
+from . import _actions as actions
+from . import run_action
+
+
+def filter_content_keys(obj: Dict[Any, Any]) -> Dict[Any, Any]:
+    """Filter out some keys when showing content.
+
+    :param obj: The object to be filtered
+    :return: The object with keys filtered out
+    """
+    return {k: v for k, v in obj.items() if not k.startswith("__")}
+
+
+def color_menu(colno: int, colname: str, entry: Dict[str, Any]) -> Tuple[int, int]:
+    # pylint: disable=unused-argument
+    """Color the menu.
+
+    :param colno: Column number
+    :param colname: Column name
+    :param entry: Column value
+    :return: Tuple int used to color the menu
```

Comment:
I don't understand what is this "tuple int" and how does it color the menu? Is it another incarnation of the same constant from https://github.com/ansible/ansible-navigator/pull/957/files#r810635617? If yes, the cure would be the same — use a named constant/label.
Also, it would be great to add some clarity to this return value description. The fact that it's a tuple is quite obvious from the type hints but the context/meaning of this data structure is not, and that's what needs to be explained better.

Reply:
Updated to: 
`:returns: Value that curses uses to set line text color`